### PR TITLE
Add accounts documentation

### PR DESCRIPTION
**Pull request by nunofgs**
This PR adds documentation for the new accounts feature that allows users to deposit value into the platform from an external source, as well as withdrawing to an external source.

R=@fixe

### CODE REVIEW

File: _accounts.md [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+
+Account type | Deposits supported? | Withdrawals supported?
+------------ | ------------------- | ----------------------
+ach          | Yes                 | Yes
+card         | Yes                 | No
+sepa         | Yes                 | Yes
+
+Please refer to our FAQ for estimated [ACH transaction times](https://support.uphold.com/hc/en-us/articles/206762103-How-to-add-and-withdraw-funds-via-bank-transfer-U-S-), [SEPA transaction times](https://support.uphold.com/hc/en-us/articles/205803186-How-to-add-and-withdraw-funds-via-bank-transfer-Europe-), [fees and limits](https://support.uphold.com/hc/en-us/articles/206118653-Transaction-Trading-Limits).
+
+## List Accounts
```
Comment (fixe) [inline _accounts.md]: Would be useful to specify which OAuth scope is required in order to make this call.

File: _accounts.md [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+### Request
+`POST https://api.uphold.com/v0/me/accounts`
+
+<br/>
+
+Required fields for all account types:
+
+Parameter | Description
+--------- | -----------------------------------------------------------------------------------------------------------------------------------
+billing   | An object with the `address`, `country` and `state` fields that must match the account details.
```
Comment (fixe) [inline _accounts.md]: Specify the format of `country` and `state`?

File: _accounts.md [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+currency  | The currency that matches the account. Uphold will automatically convert to this currency to reduce any exchange fees for the user.
+label     | The display name of the account.
+type      | The type of the account. Possible values: `card`, `sepa`.
```
Comment (fixe) [inline _accounts.md]: Missing `ACH`.

File: _accounts.md [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+```json
+{
+  "firstAmount": "0.80",
+  "secondAmount": "0.20"
+}
+```
+
+Verifies an ACH account using the two debits received on your bank account.
+<aside class="notice">
+  <strong>Important notice</strong>: An account can only be verified when it's status is `verifying`.
```
Comment (fixe) [inline _accounts.md]: Weird to see the status being hyphenated in the docs.

**Comment by nunofgs** [comment]
@fixe: Updated.

File: _accounts.md [hunk 5]
```diff
@@ -0,0 +1,326 @@
...
+`GET https://api.uphold.com/v0/me/accounts`
+<aside class="notice">Requires the `accounts:read` scope for Uphold Connect applications.</aside>
+
+### Response
+Returns an array of the current user's accounts.
+
+## Get Account Details
+
+```bash
+curl "https://api.uphold.com/v0/me/accounts/07c102bb0-f129-591b-96e4-6ea26466e012" \
```
Comment (fixe) [inline _accounts.md]: Should be a valid uuid.

**Pull request merged**